debezium-ddl-parser/src/main/java/io/debezium/antlr/AntlrDdlParser.java

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.debezium.antlr;
+
+import io.debezium.relational.TableId;
+import io.debezium.relational.Tables;
+import io.debezium.relational.ddl.AbstractDdlParser;
+import io.debezium.relational.ddl.DdlParserListener;
+import io.debezium.text.MultipleParsingExceptions;
+import org.antlr.v4.runtime.CharStream;
+import org.antlr.v4.runtime.CharStreams;
+import org.antlr.v4.runtime.CodePointCharStream;
+import org.antlr.v4.runtime.CommonTokenStream;
+import org.antlr.v4.runtime.ConsoleErrorListener;
+import org.antlr.v4.runtime.Lexer;
+import org.antlr.v4.runtime.Parser;
+import org.antlr.v4.runtime.ParserRuleContext;
+import org.antlr.v4.runtime.misc.Interval;
+import org.antlr.v4.runtime.tree.ParseTree;
+import org.antlr.v4.runtime.tree.ParseTreeWalker;
+
+/**
+ * @author Roman Kuchár <kucharrom@gmail.com>.
+ */
+public abstract class AntlrDdlParser<L extends Lexer, P extends Parser> extends AbstractDdlParser {
```

Comment:
Let's always have a comment on the class level about its purpose.

Reply:
I'm counting on that.